ReleaseNotes/ydbr120.rst

```diff
@@ -148,6 +148,139 @@ r1.20
 
 YottaDB r1.20 includes the following changes from `YottaDB r1.10 <https://github.com/YottaDB/YottaDB/releases/tag/r1.10>`_.
 
```

Comment:
With the Simple API, it occurs to me that we should not have a "Database" category, but rather "Data Management" or perhaps just "Data"

Reply:
Other:
* Alphabetize error message
* Tarball hashes
* Coipyright is 2018 
* Delete Downloads section at the end